docker-compose.env.yaml.example

```diff
@@ -3,6 +3,7 @@ DEBUG=1
 APPLY_MIGRATIONS=1
 CREATE_SUPERUSER=1
 SEED_DEVELOPMENT_DATA=0
+SET_ALLOWED_DATA_FIELDS=1
```

Comment:
This seems like an operation that should to be normally run only when setting up a new environment, so I think it would be safer to have it false in the example.

Reply:
SET_ALLOWED_DATA_FIELDS has set to 0.